README.md

```diff
@@ -16,18 +16,18 @@ _"You shall ~~not~~ PaaS"_
 ## Architecture
 
 ### Components
-- Mesos + Marathon + ZooKeeper (orchestration components)
+- Mesos + Marathon + ZooKeeper + Chronos (orchestration components)
 - Consul (K/V store, monitoring, service directory and registry)  + Registrator (automating register/ deregister)
 - HAproxy + consul-template (load balancer with dynamic config generation)
 
-![PanteraS Architecture](http://s3.amazonaws.com/easel.ly/all_easels/19186/panteras/image.jpg)
+![PanteraS Architecture](http://imageshack.com/a/img905/397/zNEiTY.jpg)
```

Comment:
You don't have to change that. I'm using http://www.easel.ly with vector graphic editor
so I can easily add more stuff